Resources/Prototypes/_Mono/Guidebook/gunnery.yml

```
@@ -0,0 +1,41 @@
+- type: guideEntry
+  id: WeaponsSystems
+  name: guide-entry-gunnery
+  text: "/ServerInfo/_Mono/Guidebook/ShipCombat.xml"
+  priority: 7
+  children:
+  - L85Guide
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)